dask/dataframe/io.py

```diff
@@ -798,9 +798,12 @@ def from_delayed(dfs, meta=None, divisions=None, prefix='from-delayed',
         An iterable of ``dask.delayed.Delayed`` objects, such as come from
         ``dask.delayed`` These comprise the individual partitions of the
         resulting dataframe.
-    $META
```

Comment:
I also thought about that, but sided with wanting a more verbose description everywhere. Could go either way though. If you want to change it I'd be +1.